tests/robot-tests/tests/libs/slack.py

```diff
@@ -71,16 +92,17 @@ def send_slack_report(env: str):
 
     print("Sent UI test statistics to #build")
 
-    client = WebClient(token=slack_bot_token)
-    print('Sending UI test report to #build')
-
-    shutil.make_archive('UI-test-report', 'zip', PATH)
-    try:
-        client.files_upload(
-            channels='#build',
-            file='UI-test-report.zip',
-            title='test-report.zip',
-        )
-    except SlackApiError as e:
-        print(f'Error uploading test report: {e}')
-    os.remove('UI-test-report.zip')
+    if should_send_report:
+        client = WebClient(token=slack_bot_token)
+        print('Sending UI test report to #build')
```

Comment:
This is a minor gripe, but I'd keep the print statements consistent in whether we make them at the start or at the end. Currently, line 93 prints at the end, and this line prints at the start. 

I think you can skip this print and instead print on it's successful completion inside the try block below.